.github/workflows/check.yml

```
@@ -13,14 +13,25 @@
   issues: write
   pull-requests: write
 
-defaults:
-  run:
-    working-directory: client
-
 jobs:
+  check-server:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    working-directory: server
```

actionlint (1.7.4)

unexpected key "working-directory" for "job" section. expected one of "concurrency", "container", "continue-on-error", "defaults", "env", "environment", "if", "name", "needs", "outputs", "permissions", "runs-on", "secrets", "services", "steps", "strategy", "timeout-minutes", "uses", "with"

(syntax-check)